msr/msre/multiphysics_core_model/steady_state/neu.i

```diff
@@ -0,0 +1,654 @@
+  []
+  [fission_rate]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [Absorption_rate]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [Leakage_rate]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [c1]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 0.000
+    block = 'core lower_plenum upper_plenum down_comer riser pump elbow'
+  []
+  [c2]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 0.000
+    block = 'core lower_plenum upper_plenum down_comer riser pump elbow'
+  []
+  [c3]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 0.000
+    block = 'core lower_plenum upper_plenum down_comer riser pump elbow'
+  []
+  [c4]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 0.000
+    block = 'core lower_plenum upper_plenum down_comer riser pump elbow'
+  []
+  [c5]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 0.000
+    block = 'core lower_plenum upper_plenum down_comer riser pump elbow'
+  []
+  [c6]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 0.000
+    block = 'core lower_plenum upper_plenum down_comer riser pump elbow'
+  []
+  [dnp]
+    order = CONSTANT
+    family = MONOMIAL
+    components = 6
+    block = 'core lower_plenum upper_plenum down_comer riser pump elbow'
+  []
+
+  [ad_U235]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 9.692763E-05
+  []
+  [ad_U238]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 1.967924E-04
+  []
+  [ad_Be9]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 9.496943E-03
+  []
+  [ad_Li7]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 2.121311E-02
+  []
+  [ad_F9]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 4.790899E-02
+  []
+  [ad_Zr90]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 8.395505E-04
+  []
+  [ad_Zr91]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 1.830855E-04
+  []
+  [ad_Zr92]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 2.798495E-04
+  []
+  [ad_Zr94]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 2.836027E-04
+  []
+  [ad_Zr96]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 4.568967E-05
+  []
+  [ad_C12]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = 7.247622E-02
+    block = 'core core_barrel'
+  []
+[]
+
+[AuxKernels]
+  [build_dnp]
+    type = BuildArrayVariableAux
+    variable = dnp
+    component_variables = 'c1 c2 c3 c4 c5 c6'
+    execute_on = 'initial timestep_begin final'
+  []
+  [update_ad_f_U235]
+    type = ParsedAux
+    block = 'lower_plenum upper_plenum down_comer riser pump elbow'
+    variable = ad_U235
+    args = 'T_salt'
+    function = '9.692763E-05*(1.0-0.4798*(T_salt-${T_Salt_initial})/${Salt_Density_initial})'
+    execute_on = 'INITIAL timestep_end'
+  []
```

Comment:
args and function have been deprecated for two years

Reply:
Updated.